src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1806,6 +1806,9 @@ public static bool IsInstanceContext(this SyntaxTree syntaxTree, SyntaxToken tar
 
             while (enclosingSymbol is IMethodSymbol method && (method.MethodKind == MethodKind.LocalFunction || method.MethodKind == MethodKind.AnonymousFunction))
             {
+                if (method.IsStatic)
+                    return false;
```

Comment:
IIRC even trivial unbraced ifs are discouraged in the codebase.

Reply:
@YairHalberstadt Yes, we do always do braced ifs. From my perspective this is the only thing blocking this from merge.

Reply:
Braces added